src/styles/_Footer.scss

```diff
@@ -7,6 +7,8 @@
   flex: none;
   display: flex;
   font-size: 1.1em;
+  position: absolute;
+  bottom: 0;
```

Comment:
The footer has to be below all content. 
See the results of this on the homepage

![image](https://user-images.githubusercontent.com/5168796/53516266-b8abc300-3a80-11e9-8b93-21ad28a61f94.png)